code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -70,8 +70,8 @@
 		to_chat(user, "<span class='info'>[M]'s biological structure is too complex for the health analyzer.")
 		return
 
-	user.visible_message(span_notice("[user] analyzes [M]'s vitals."), \
-						span_notice("You analyze [M]'s vitals."))
+	user.visible_message(span_notice("[user] analyzes [M]'s vitals."))
+	balloon_alert(user, "analyzing vitals")
```

Comment:
I don't think this needs a balloon alert since it already has some other sort of input (that being the huge health scan in the chat, someday maybe a popup or something)

Reply:
My issue with it is that sometimes you try to scan someone, think it worked and move your attention to the chat only to see that you missed by a pixel and it didn't work.
Now if the target moved you need to keep running after them and attempting again.
Either way I can remove it if you don't think this is a good use for balloon alert.

Reply:
That's reasonable enough